Documentation/helm-values.rst

```diff
@@ -1197,6 +1213,14 @@
      - Configure BPF socket operations configuration
      - object
      - ``{"enabled":false}``
+   * - startupProbe.failureThreshold
+     - failure threshold of startup probe. 105 x 2s translates to the old behaviour of the readiness probe (120s delay + 30 x 3s)
+     - int
+     - ``105``
+   * - startupProbe.periodSeconds
+     - interval between checks of the startup probe
+     - int
+     - ``2``
```

Comment:
Have these changes been included by mistake?

Reply:
:+1: Ok, makes sense. That information should ideally be in the commit message though :-)